From: Maria Valentina Marin <marivalenm@gmail.com>
Date: Wed, 25 Feb 2015 10:05:57 +0100
Subject: [Pkg-alsa-devel] Bug#779182: alsa-lib: please make the build reproducible

Source: alsa-lib
Version: 1.0.28-1
Severity: wishlist
Tags: patch
User: reproducible-builds@lists.alioth.debian.org
Usertags: timestamps

Hi!

While working on the “reproducible builds” effort [1], we have noticed
that alsa-lib could not be built reproducibly.

The attached patch removes extra timestamps in the doxygen documentation from
the build system. Once applied, alsa-lib can be built reproducibly in our
current experimental framework.

Thanks, akira

 [1]: https://wiki.debian.org/ReproducibleBuilds


Index: alsa-lib-1.0.28/doc/doxygen.cfg.in
===================================================================
--- alsa-lib-1.0.28.orig/doc/doxygen.cfg.in
+++ alsa-lib-1.0.28/doc/doxygen.cfg.in
@@ -3,7 +3,7 @@ OUTPUT_DIRECTORY = doxygen
 GENERATE_LATEX   = NO
 GENERATE_MAN     = NO
 GENERATE_RTF	 = NO
-
+HTML_TIMESTAMP   = NO
 CASE_SENSE_NAMES = NO
 INPUT            = @top_srcdir@/doc/index.doxygen \
 		   @top_srcdir@/include/asoundlib.h \
